modules/consensus/synchronize.go

```diff
@@ -91,8 +107,13 @@ func (cs *ConsensusSet) threadedReceiveBlocks(conn modules.PeerConn) error {
 
 		// Integrate the blocks into the consensus set.
 		for _, block := range newBlocks {
-			acceptErr := cs.AcceptBlock(block)
-
+			// Call managedAcceptBlock instead of AcceptBlock so as not to broadcast
+			// every block.
+			acceptErr := cs.managedAcceptBlock(block)
+			// Only broadcast the last block if a block was accepted.
+			if acceptErr == nil {
+				chainExtended = true
+			}
```

Comment:
Should this be placed after the
``` go
// ErrNonExtendingBlock must be ignored until headers-first block
// sharing is implemented, block already in database should also be
// ignored.
if acceptErr == modules.ErrNonExtendingBlock || acceptErr == modules.ErrBlockKnown {
	acceptErr = nil
}
```
block so that a non-extending and known blocks are still broadcast?